index.js

```diff
@@ -1,6 +1,10 @@
 const core = require('@actions/core');
 const deploy = require('./deploy');
 
+function getBooleanInput(name) {
+  return core.getInput(name) === 'true';
```

Comment:
Could you add lower case conversion to prevent case mismatch?

Reply:
done